providers/recurly.go

```diff
@@ -0,0 +1,23 @@
+package providers
+
+const Recurly Provider = "recurly"
+
+func init() {
+	SetInfo(Recurly, ProviderInfo{
+		AuthType: Basic,
+		BaseURL:  "https://v3.recurly.com",
+		Support: Support{
+			BulkWrite: BulkWriteSupport{
+				Insert: false,
+				Update: false,
+				Upsert: false,
+				Delete: false,
+			},
+			Proxy:     false,
+			Read:      false,
+			Subscribe: false,
+			Write:     false,
+		},
+		PostAuthInfoNeeded: true,
```

Comment:
@Beadko `PostAuthInfoNeeded` is used to indicate whether Ampersand needs to make an API call to an additional endpoint after getting a token to retrieve info such as `workspaceRef` or server region, etc that we need to know to make subsequent API calls. I don't see any mention of an endpoint like this in the Slab doc, and this kind of an endpoint like this.